packages/m2m-gateway/test/mockUtils.ts

```diff
@@ -151,6 +156,35 @@ export function getMockedApiEservice(): WithMetadata<catalogApi.EService> {
   };
 }
 
+export function getMockedApiEserviceDescriptor(): WithMetadata<catalogApi.EServiceDescriptor> {
+  return {
+    data: {
+      id: generateId(),
+      version: generateMock(z.string()),
+      description: generateMock(z.string()),
```

Comment:
```suggestion
      description: generateMock(z.string().length(10)),
```

Same as the eservice mock